pkgs/development/compilers/chicken/5/overrides.nix

```diff
@@ -35,33 +35,46 @@ in {
   freetype = addToBuildInputsWithPkgConfig pkgs.freetype;
   fuse = addToBuildInputsWithPkgConfig pkgs.fuse;
   # git = addToBuildInputsWithPkgConfig pkgs.libgit2;
+  gl-utils = addPkgConfig;
   glfw3 = addToBuildInputsWithPkgConfig pkgs.glfw3;
+  glls = addPkgConfig;
+  iconv = addToBuildInputs (lib.optional stdenv.isDarwin pkgs.libiconv);
   icu = addToBuildInputsWithPkgConfig pkgs.icu;
   imlib2 = addToBuildInputsWithPkgConfig pkgs.imlib2;
-  lazy-ffi = old: (brokenOnDarwin old) // (addToBuildInputs pkgs.libffi old);
+  lazy-ffi = old:
+    # fatal error: 'ffi/ffi.h' file not found
```

Comment:
libffi should provide it

Reply:
libffi only provides `ffi.h` directly in the include folder without the `ffi/`  subfolder. The path would need to be patched.

Reply:
Yeah, it should work with

```nix
  lazy-ffi = old:
    (addToBuildInputs pkgs.libffi old) // {
      postPatch = ''
        substituteInPlace lazy-ffi.scm \
          --replace "ffi/ffi.h" "ffi.h"
      '';
    };
```